theano/tensor/tests/test_opt.py

```diff
@@ -1941,6 +1941,44 @@ def test_local_subtensor_remove_broadcastable_index():
     f2(xn)
 
 
+def test_subtensor_inc_subtensor():
```

Comment:
I don't remember exactly, I guess I thought `v` was a vector, not a tensor3.
In any case, running the function with values so that DebugMode can check the correctness of the optimization should be necessary.